src/Orchard.Specs/Enumeration.feature

```diff
@@ -56,85 +56,87 @@ Scenario: Creating and using Enumeration fields
     When I go to "Admin/ContentTypes/Edit/Event"
         And I fill in 
-            | name                                        | value       |
-            | Fields[0].EnumerationFieldSettings.ListMode | Radiobutton |
+            | name                                               | value             |
+            | Fields[Location].EnumerationFieldSettings.ListMode | Radio button list |
         And I hit "Save"
         And I go to "Admin/Contents/Create/Event"
     Then I should see "input id=\"Event_Location_Value\" name=\"Event.Location.Value\" type=\"radio\""
     
     # The List Mode Listbox
     When I go to "Admin/ContentTypes/Edit/Event"
         And I fill in 
-            | name                                        | value   |
-            | Fields[0].EnumerationFieldSettings.ListMode | Listbox |
+            | name                                               | value   |
+            | Fields[Location].EnumerationFieldSettings.ListMode | Listbox |
         And I hit "Save"
         And I go to "Admin/Contents/Create/Event"
     Then I should see "select id=\"Event_Location_SelectedValues\" multiple=\"multiple\" name=\"Event.Location.SelectedValues\""
     
     # The List Mode Checkbox
     When I go to "Admin/ContentTypes/Edit/Event"
         And I fill in 
-            | name                                        | value    |
-            | Fields[0].EnumerationFieldSettings.ListMode | Checkbox |
+            | name                                               | value    |
+            | Fields[Location].EnumerationFieldSettings.ListMode | Checkbox |
         And I hit "Save"
         And I go to "Admin/Contents/Create/Event"
     Then I should see "input type=\"checkbox\" name=\"Event.Location.SelectedValues\""
     
     # The value should be required
     When I go to "Admin/ContentTypes/Edit/Event"
         And I fill in 
-            | name                                        | value |
-            | Fields[0].EnumerationFieldSettings.Required | true  |
+            | name                                               | value |
+            | Fields[Location].EnumerationFieldSettings.Required | true  |
         And I hit "Save"
         And I go to "Admin/Contents/Create/Event"
         And I hit "Save Draft"
     Then I should see "The Location field is required."
     
-    # The default value should be proposed on creation
-    When I go to "Admin/ContentTypes/Edit/Event"
-        And I fill in 
-            | name                                            | value    |
-            | Fields[0].EnumerationFieldSettings.Options      | Seattle  |
-            | Fields[0].EnumerationFieldSettings.ListMode     | Dropdown |
-            | Fields[0].EnumerationFieldSettings.DefaultValue | Seattle  |
-        And I hit "Save"
-        And I go to "Admin/Contents/Create/Event"
-    Then I should see "selected=\"selected">Seattle"
+   # # The default value should be proposed on creation
```

Comment:
Why is his one commented?

Reply:
This comment is test about *Default value should be proposed on creation*
But when I go to "Admin/ContentTypes/Edit/Event" and look at **Numeric Field**. It has no text box of Default value for input data.